### ISSUE TITLE
different colour for note head inters

### ISSUE THREAD

**Issue by Bacchushlg** [Audiveris/audiveris#73]
note heads inters are actually displayed in orange. This is mostly no problem, but can be in case of heads in a measure with rose background (error in rhythm).
![image](https://user-images.githubusercontent.com/33522630/36255380-17c2cb3c-124f-11e8-9eb2-e04d8375ed6f.png)
(There is a whole note detected instead of the f-clef)

I propose to use a darker colour, maybe simply blue, which is actually used by accidentals, but I think it's problem to use the same colour for both.

**Comment by hbitteur** [comment]
There are several aspects to take into account here:
1. A given `shape `(or group of shapes) is **assigned** a given color.
2. A given inter is displayed in **red** if in abnormal state (for example a slur lacking head connection). Otherwise it uses the registered color for its shape (see aspect 1 above).
3. In the default input mode, inter shape color is actually displayed with a **opacity** value which grows in line with its grade: a poor-quality inter is nearly translucid while a very good-quality inter is fully opaque.
This last aspect might be the reason for your (poor) whole note to not be very visible. This opacity aspect does not apply to the other modes (combined:green and output:black).

Regarding the shape/color mapping, there is still a `ShapeColorChooser `class, no used actually, that allowed the user to choose his own colors,
Right now the mapping is **hard-coded**. You can see color assignments in `Shape `and `ShapeSet `classes.
Colors have been chosen to clearly display the difference between often conflicting inters, such as `NOTEHEAD_BLACK` versus `NOTEHEAD_VOID`.

In short, modifying shape/color mapping is a bit touchy.

**Comment by Bacchushlg** [comment]
![image](https://user-images.githubusercontent.com/33522630/36262319-625e5fb8-1267-11e8-85e6-7cb152a824ca.png)

I tried a couple of colours. This seems to be not too bad:
- note heads in dark green (0x008844) (including whole notes and aug. dots)
- stems in brown (806040)

Good idea?

**Comment by hbitteur** [comment]
@Bacchushlg I updated Shape and ShapeSet classes with your versions as commit a09148a928c2aa4900c7a24c1eeb8bf4f746ff41
Feel free to update these classes to come up with a final clean color scheme, I just invited you to the project as a collaborator

**Comment by hbitteur** [comment]
@Bacchushlg 
To clean up this handling of item colors, I would suggest to get rid of color "literals" that are today hard-coded in Shape and ShapeSet classes. We could replace these literals by logical constants as some of them are already gathered in the Colors class within `org.audiveris.omr.ui` package.

For example, continuing with your interesting proposals of "families", we could simply define a few family colors in Colors class, for example `Color FAMILY_XYZ = new Color(litteral);` where we should carefully choose the "XYZ" parts.

I think that such "families" are larger than the current "ShapeSet" definitions, in fact a given family might gather several ShapeSet's. A ShapeSet is today used to govern the Shape palette (note there is by design one shape set per ShapeSet, hence the name) and also for some algorithmic tests. My feeling here is that a family is not exactly a ShapeSet or a group of ShapeSet's, so it would be safer to keep them rather decoupled.

**Comment by Bacchushlg** [comment]
You run in open doors ("du rennst offene Türen ein" - in German).
This was exactly my idea, provide a set of colours that are then referenced by the ShapeSets (and e. overwritten by some Shapes).
Unfortunately I am a C-Programmer. The easiest way there would be a couple of constants holdings these colour values. The Shapesets then only reference the constants.

Could you provide a skeleton code, so I could do the rest of the work?

**Comment by hbitteur** [comment]
In French: "_tu enfonces une porte ouverte_"

The easiest way for both of us:
1. You work on a creating the list of "family" names, each with its specific color litteral
2. I then use it to update the Java code accordingly in Colors, Shape and ShapeSet classes. Easy.

About lyrics color: I created a specific shape (LYRICS, with its own color on line 379) beside TEXT shape, specifically to manually create the lyrics lines. But I made a mistake, WordInter (superclass of LyricItemInter) is still assigned the TEXT shape. And since shape is used to determine color, this is the reason why you cannot obtain for lyrics a color different of the basic text. My mistake, sorry. I'll fix it ASAP.

**Comment by hbitteur** [comment]
Done via commit ce1ed323aeeafa407adc7a14a60d16be99512031
Now lyrics appear in a kind of green color, and are visually very different from other text items.
